### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/tutorial/sql-databases.md`

### PR DESCRIPTION
**Pull request by dnfto**
This PR translates docs/ko/docs/tutorial/sql-databases.md in Korean.
related: #3167

### CODE REVIEW

File: docs/ko/docs/tutorial/sql-databases.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+
+**FastAPI**에서 SQL(관계형) 데이터베이스 사용은 필수가 아닙니다. 여러분이 원하는 **어떤 데이터베이스든** 사용할 수 있습니다.
+
+여기서는 <a href="https://sqlmodel.tiangolo.com/" class="external-link" target="_blank">SQLModel</a>을 사용하는 예제를 살펴보겠습니다.
+
+**SQLModel**은 <a href="https://www.sqlalchemy.org/" class="external-link" target="_blank">SQLAlchemy</a>와 Pydantic을 기반으로 구축되었습니다.SQLModel은 **SQL 데이터베이스**를 사용하는 FastAPI 애플리케이션에 완벽히 어울리도록 **FastAPI**의 제작자가 설계한 도구입니다.
+
+/// 팁
+
+다른 SQL 또는 NoSQL 데이터베이스 라이브러리를 사용할 수도 있습니다 (일부는 <abbr title="객체 관계 매퍼(Object Relational Mapper), SQL 테이블을 나타내는 클래스를 제공하고 테이블의 행을 인스턴스로 표현하는 라이브러리를 지칭하는 용어">"ORM"</abbr>이라고도 불립니다), FastAPI는 특정 라이브러리의 사용을 강요하지 않습니다. 😎
```
Comment (guspan-tanadi) [inline docs/ko/docs/tutorial/sql-databases.md:11]: Hi, the box can have caption
`/// tip | 팁`
so then box will display to the page

File: docs/ko/docs/tutorial/sql-databases.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+
+#### `HeroCreate` - hero 생성용 *데이터 모델*
+
+이제 `HeroCreate` 모델을 생성합니다. 이 모델은 클라이언트로부터 받은 데이터를 **검증**하는 역할을 합니다.
+
+`HeroCreate`는 `HeroBase와` 동일한 필드를 가지며, 추가로 `secret_name을` 포함합니다.
+
+클라이언트가 **새 hero을 생성**할 때 `secret_name`을 보내고, 이는 데이터베이스에 저장되지만, 해당 비밀 이름은 API를 통해 클라이언트에게 반환되지 않습니다.
+
+/// 팁
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/sql-databases.md:251]:
```suggestion
/// tip | 팁
```

File: docs/ko/docs/tutorial/sql-databases.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+
+이전 애플리케이션에서는 **hero를 수정**할 방법이 없었지만, 이제 **다중 모델**을 통해 수정 기능을 추가할 수 있습니다. 🎉
+
+`HeroUpdate` *데이터 모델*은 약간 특별한데, 새 hero을 생성할 때 필요한 **모든 동일한 필드**를 가지지만, 모든 필드가 **선택적**(기본값이 있음)입니다. 이렇게 하면 hero을 수정할 때 수정하려는 필드만 보낼 수 있습니다.
+
+모든 **필드가 변경되기** 때문에(타입이 `None`을 포함하고, 기본값이 `None`으로 설정됨), 모든 필드를 **다시 선언**해야 합니다.
+
+엄밀히 말하면 `HeroBase`를 상속할 필요는 없습니다. 모든 필드를 다시 선언하기 때문입니다. 일관성을 위해 상속을 유지하긴 했지만, 필수는 아닙니다. 이는 개인적인 취향의 문제입니다. 🤷
+
+`HeroUpdate의 필드는 다음과 같습니다:
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/sql-databases.md:277]:
```suggestion
`HeroUpdate`의 필드는 다음과 같습니다:
```
Missed backtick

File: docs/ko/docs/tutorial/sql-databases.md [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+
+요청에서 `HeroCreate` *데이터 모델*을 받아 이를 기반으로 `Hero` *테이블 모델*을 생성합니다.
+
+이 새 *테이블 모델* `Hero`는 클라이언트에서 보낸 필드를 가지며, 데이터베이스에서 생성된 `id`도 포함합니다.
+
+그런 다음 함수를 통해 동일한 *테이블 모델* `Hero`를 반환합니다. 하지만 `response_model`로 `HeroPublic` *데이터 모델*을 선언했기 때문에, **FastAPI**는 `HeroPublic`을 사용하여 데이터를 검증하고 직렬화합니다.
+
+{* ../../docs_src/sql_databases/tutorial002_an_py310.py ln[56:62] hl[56:58] *}
+
+/// 팁
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/sql-databases.md:297]:
```suggestion
/// 팁 | tip
```

**Review by hard-coders — COMMENTED** [review]
Great job! I've reviewed it, so please check it out 😄

**Comment by GeumBinLee** [comment]
Thanks for your review! It helped me a lot 😀 I reflected the modifications.

**Comment by JaeHyuckSa** [comment]
LGTM 👍

**Review by JaeHyuckSa — APPROVED** [review]

**Review by hard-coders — APPROVED** [review]
LGTM 🚀 
It's unimportant, but it's better for other reviewers if you "resolve" reviewed comments.

**Comment by alejsdev** [comment]
Thanks for your help! @GeumBinLee :rocket: 
And thanks for your review @JaeHyuckSa :nerd_face:

**Comment by alejsdev** [comment]
Thanks for your help! @GeumBinLee :rocket: 
And thanks for your review @hard-coders :nerd_face:

**Pull request merged**